Makefile

```diff
@@ -0,0 +1,23 @@
+BIND_DIR := build
+MKDIR_P = mkdir -p
+RM = rm -rf
+
+default: build-all
+
+build-jessie:
```

Comment:
Yeah, but it would be good to use pattern targets rather than hardcoding the list, then we can just modify a list variable to add new ones (like when debian 9 is released next month)